test/integrations/destinations/facebook_pixel/processor/configLevelFeaturesTestData.ts

```diff
@@ -0,0 +1,483 @@
+import { VERSION } from '../../../../../src/v0/destinations/facebook_pixel/config';
+import { overrideDestination, generateTrackPayload } from '../../../testUtils';
+
+const commonDestination = {
+  ID: '12335',
+  Name: 'sample-destination',
+  DestinationDefinition: {
+    ID: '123',
+    Name: 'facebook_pixel',
+    DisplayName: 'Facebook Pixel',
+    Config: {},
+  },
+  WorkspaceID: '123',
+  Transformations: [],
+  Config: {
+    limitedDataUSage: true,
+    blacklistPiiProperties: [
+      {
+        blacklistPiiProperties: '',
+        blacklistPiiHash: false,
+      },
+    ],
+    accessToken: '09876',
+    pixelId: 'dummyPixelId',
+    eventsToEvents: [
+      {
+        from: 'ABC Started',
+        to: 'InitiateCheckout',
+      },
+    ],
+    eventCustomProperties: [
+      {
+        eventCustomProperties: '',
+      },
+    ],
+    valueFieldIdentifier: '',
+    advancedMapping: false,
+    whitelistPiiProperties: [
+      {
+        whitelistPiiProperties: 'email',
+      },
+    ],
+    categoryToContent: [
+      {
+        from: 'clothing',
+        to: 'newClothing',
+      },
+    ],
+  },
+  Enabled: true,
+};
+
+const commonUserTraits = {
+  email: 'abc@gmail.com',
+  anonymousId: 'c82cbdff-e5be-4009-ac78-cdeea09ab4b1',
+  event_id: '12345',
+};
+
+const piiPropertiesForAllowDeny = {
+  email: 'abc@gmail.com',
+  anonymousId: 'c82cbdff-e5be-4009-ac78-cdeea09ab4b1',
+  event_id: '12345',
+  firstName: 'John',
+  lastName: 'Doe',
+  whitelistProp1: 'val1',
+  blacklistProp2: 'val2',
+  blacklistProp3: 'val3',
+  category: 'dummy',
+  quantity: 10,
+  value: 100,
+  product_id: '12345',
+};
+
+const commonPropertiesWithoutProductArray = {
+  category: 'dummy',
+  quantity: 10,
+  value: 100,
+  product_id: '12345',
+};
+
+const commonTimestamp = new Date('2023-10-14');
+
+export const configLevelFeaturesTestData = [
+  {
+    name: 'facebook_pixel',
+    description: 'config feature : limitedDataUSage switched on',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: generateTrackPayload({
+              event: 'product list viewed',
+              properties: commonPropertiesWithoutProductArray,
+              context: {
+                traits: commonUserTraits,
+                dataProcessingOptions: ['val1', 'val2', 'val3'],
+              },
+              timestamp: commonTimestamp,
+            }),
+            destination: commonDestination,
+          },
+        ],
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            output: {
```

Comment:
https://github.com/rudderlabs/rudder-transformer/blob/511cfacd3b887bf6c4f5f16b9e6b3ed9a4dc6bd8/test/integrations/destinations/klaviyo/processor/groupTestData.ts#L77

use this function to reduce boiler plate efforts

Reply:
used